### DISCUSSION TITLE
Question For Interstitial Ads Adding!

### DISCUSSION THREAD

**Discussion by Orbit-mm** [Sketchware-Pro/Sketchware-Pro#733] [Q&A]
Pleas Explain about adding Interstitial Ads in This version.Please 🥺

**Comment by DevVigilante** [comment]
> Pleas Explain about adding Interstitial Ads in This version.Please 🥺

Do you have active admob id?

**Reply by Orbit-mm** [reply]
Yes I have and my id is been approved by google for 1 yr ago .

**Reply by Orbit-mm** [reply]
I can activate the banner but can't Interstitial Ads to show.

**Reply by DevVigilante** [reply]
I successfully showed read ads in my app contact me personally at https://t.me/Dev7578

**Comment by DevVigilante** [comment]
I successfully approved my app in admob by publishing it on Amazon app store.My created by sketchware contact me at https://t.me/Dev7578 and send swb file don't share it here
